config.go

```diff
@@ -72,15 +72,15 @@ func (c *Config) AddMembershipListener(handler cluster.MembershipStateChangeHand
 	return id
 }
 
-// AddNearCacheConfig adds a near cache configuration.
-func (c *Config) AddNearCacheConfig(cfg nearcache.Config) {
+// AddNearCache adds a near cache configuration.
+func (c *Config) AddNearCache(cfg nearcache.Config) {
```

Comment:
This is changing the public API and do not conform to TDD.

Reply:
Yes, I will update the public API TDD and notify the reviewers.